docs/users/developing.md

```diff
@@ -335,9 +335,10 @@ These checks will use the most recent versions of each linter.
 To ensure that changes follow these standards, you can install and run these tools locally:
 
 ```shell
-conda install black isort -c conda-forge
+mamba install black isort flake8 -c conda-forge
```

Comment:
(not blocking) Do we also need flake8-bugbear?

Reply:
Strictly speaking, plugins should be installed here, but I don't think it's too important and I'd rather keep the pre-commit config as the sole source of truth. With the toolkit's config, there shouldn't be many cases in which the base package passes but a plugin throws an error